cluster/gce/upgrade.sh

```diff
@@ -447,6 +447,27 @@ if [[ "${master_upgrade}" == "false" ]] && [[ "${node_upgrade}" == "false" ]]; t
   exit 1
 fi
 
+# prompt if etcd storage media type isn't set
+if [[ -z "${STORAGE_MEDIA_TYPE:-}" ]]; then
```

Comment:
if `"${STORAGE_BACKEND}" == "etcd2"`, then media type is forced to `application/json`. Not sure we want to nag them into setting it explicitly (if we do, I guess I'm ok with it).